src/e3/testsuite/report/gaia.py

```diff
@@ -63,6 +63,14 @@ def dump_gaia_report(testsuite: TestsuiteCore, output_dir: str) -> None:
         which to generate the report.
     :param output_dir: Directory in which to emit the report.
     """
+    with open(
+        os.path.join(output_dir, "discs"), "w", encoding="utf-8"
+    ) as discs_fd:
+        if testsuite.env.discs is not None:
+            discs_fd.write(" ".join([
+                disc for disc in testsuite.env.discs if disc is not None
+            ]))
+
```

Comment:
`testsuite.env.discs` does not necessarily exist, so this may crash at runtime. What about writing this file iff `testsuite.env.discs` 1) exists 2) is an iterable of strings? Could you add testcases for each case? Also, please open a ticket to host this work. :-) Thanks Ghjuvan!